tests/installation/partitioning/guided_setup.pm

```diff
@@ -19,7 +19,8 @@ use scheduler 'get_test_suite_data';
 sub run {
     my $test_data    = get_test_suite_data()->{guided_partitioning};
     my $guided_setup = $testapi::distri->get_guided_partitioner();
-
+    # Select disks to use, if multiple disks are available
+    $guided_setup->select_disks_to_use(@{$test_data->{disks}}) if $test_data->{disks};
```

Comment:
I'm using this convention to use 'setup_something' to indicate that we click next at the end, but it is just an idea, this is perfectly fine. wdyt?

Reply:
The difference here is that the whole screen might be missing in case of single disk, so here it's slightly different. In your case we would always have the pages, but might leave the defaults.
We can introduce separate test module where this action is expected to be called, but I think we agreed to follow test data driven approach, so from looking at test data we should know what to expect. Is that what you think? Or you mean just to call the method to e.g. `setup_disks_to_use` ?

Reply:
yes, just the naming by convention if we like it.

Reply:
I don't think that `setup` prefix always expresses the action better than other verbs like set/select/.. But for this one I can rename it.

Reply:
yes, I don't know better name to express that, 'process' (the page) or 'fill_in'. you don't need to change it, just an idea.